opds.py

```diff
@@ -302,6 +302,9 @@ def active_licensepool_for(cls, work):
                 active_license_pool = p
         return active_license_pool
 
+    def sort_works_for_groups_feed(self, works, **kwargs):
```

Comment:
This is a hook that, by default, doesn't impact the works at all. It is overwritten in `StaticFeedAnnotator`